framework/.eslintrc.json

```diff
@@ -47,19 +47,11 @@
 			}
 		],
 		"callback-return": "error",
-		"generator-star-spacing": [
```

Comment:
Please, confirm this was on purpose since the commit just points to the other rule.

Reply:
yes it is removed since it's not used